backend/models/stock.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require ('mongoose')
+const stockSchema = new mongoose.Schema({
+    name:{
+        type: String,
+        required:true
+    },
+    price:{
+        type: Number,
+        required: true
+    },
+
+    monthlyPrice:{
+        type: {}
```

Comment:
Is this  an array ? If so, why dont we specify it as `[Number]`

Reply:
It's a object but kind of weird time-series data, I will changed it to `Object`
https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY&symbol=AAPL&interval=5min&apikey=FZD9JFSVI7D6HY5E